src/controllers/transfer/transfer.ts

```diff
@@ -280,8 +280,16 @@ export class TransferController extends EventEmitter {
       ? FEE_COLLECTOR.toLowerCase()
       : this.recipientAddress.toLowerCase()
 
+    if(parseFloat(this.amount)< 1/10**this.selectedToken.decimals)
+      return this.emitError({
+        level: 'major',
+        message:
+          'Token amount too low.',
+        error: new Error('transfer: user requested token transfer amount too low.')
+      })
+
```

Comment:
The idea is good, but this is not a good place to validate the amount. The amount should only be validated by `validateSendTransferAmount` so the user receives immediate feedback.